airflow/contrib/kubernetes/pod_launcher.py

```diff
@@ -122,6 +122,8 @@ def _monitor_pod(self, pod, get_logs):
         if get_logs:
             logs = self.read_pod_logs(pod)
             for line in logs:
+                if isinstance(line, bytes):
+                    line = line.decode('utf-8').strip()
```

Comment:
Is this a related change?

Reply:
Yes, by default the pod_launcher return the pod log as a binary data, 
The ANSI coloring for the kubernetes operator worked only after this change.
There is no reason to stdout log as a binary string (aka b'log')

Reply:
Given all this is loop is doing is reading in from one place and writing out somewhere else it should be fine as binary.

Please explain why this change is needed in more detail. What happened without it?

Reply:
I revert this change